Voter/ReportVoterInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Staffim\RollbarBundle\Voter;
+
+interface ReportVoterInterface
+{
+    public function vote($exception);
```

Comment:
Probably it reasonable to add typehint ``Exception`` for ``$exception`` argument